src/index.ts

```diff
@@ -78,6 +78,8 @@ export type {
   DropdownItemProps,
   DropdownMenuProps
 } from './Dropdown'
+export {DropdownButton} from './DropdownMenu/DropdownButton'
+export {DropdownMenu} from './DropdownMenu/DropdownMenu'
```

Comment:
```suggestion
export {DropdownMenu} from './DropdownMenu'
export type {DropdownMenuProps} from './DropdownMenu'
```

Reply:
I think this is a good change, however there is a conflict with the types exported above: https://github.com/primer/components/blob/0b5678143c2c4be93cecc955bf6547b775163595/src/index.ts#L74-L80

To avoid the conflict, I'd say we just export the new menu and button without the prop types for now

Reply:
Works for me 👍 Maybe we should leave a comment about why they're not exported